src/mass_properties/mass_properties_convex_polygon.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(dead_code)] // TODO: remove this
```

Comment:
I'm not sure why it was necessary ; I'm thinking some feature flag or just a temporary addition while the code wasn't used :thinking:

Reply:
Yeah, that sounds likely.